run_kraken.py

```diff
@@ -12,16 +12,24 @@
 import kraken.invoke.command as runcommand
 import kraken.node_actions.common_node_functions as nodeaction
 from kraken.node_actions.aws_node_scenarios import aws_node_scenarios
+from kraken.node_actions.general_cloud_node_scenarios import general_node_scenarios
 from kraken.node_actions.gcp_node_scenarios import gcp_node_scenarios
 import kraken.time_actions.common_time_functions as time_actions
 
 
 # Get the node scenarios object of specfied cloud type
 def get_node_scenario_object(node_scenario):
+    if "cloud_type" not in node_scenario.keys() or node_scenario['cloud_type'] == "generic":
+        return general_node_scenarios()
     if node_scenario['cloud_type'] == 'aws':
         return aws_node_scenarios()
     elif node_scenario['cloud_type'] == 'gcp':
         return gcp_node_scenarios()
+    else:
```

Comment:
Still using arbitrary values as 'generic'.  Missed a file on the last commit?

Reply:
Sorry about that, somehow missed the main file in my last commit! Should be there now